docs/style-guides/js.md

```diff
@@ -0,0 +1,252 @@
+# origin-web-console JS style guide
+
+A reasonable approach to assisting in style consistency in `openshift-web-console`
+inspired by guides such as the style guide published by [airbnb](https://github.com/airbnb/javascript).
+
+## ESLint
+
+Run `grunt eslint` to check code for errors and style discrepancies.
+
+## Examples
+
+
+### Variables
+
+Group your variable names but use individual `var` statements:
```

Comment:
I don't have a strong opinion on this, but a lot of our current code doesn't do it. Unless we feel strongly about something, I think we should take the current style we're using as default. I will say it's something useful to declare the variable near where it's used. Would this apply to giant controller functions, too?